contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -574,7 +536,7 @@ mod tests {
         }
 
         // But voter1 can
-        let env = mock_env(voter1, &[]);
+        let env = mock_env("voter0001", &[]);
```

Comment:
Good to define these strings as constants in `mod test`, and then use those same constants in the tests as well as in the setup.

Reply:
OK. Good idea.